<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetTransform</class>
 <widget class="QWidget" name="WidgetTransform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transform Object</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelObject">
       <property name="text">
        <string>Object:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLabel" name="labelObjectName">
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pushButtonPick">
       <property name="text">
        <string>Pick</string>
       </property>
      </widget>
     </item>
     
     <item row="1" column="0">
      <widget class="QLabel" name="labelPosition">
       <property name="text">
        <string>Position</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="spinBoxPosX">
       <property name="minimum">
        <double>-10000.0</double>
       </property>
       <property name="maximum">
        <double>10000.0</double>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="spinBoxPosY">
       <property name="minimum">
        <double>-10000.0</double>
       </property>
       <property name="maximum">
        <double>10000.0</double>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QDoubleSpinBox" name="spinBoxPosZ">
       <property name="minimum">
        <double>-10000.0</double>
       </property>
       <property name="maximum">
        <double>10000.0</double>
       </property>
      </widget>
     </item>

     <item row="2" column="0">
      <widget class="QLabel" name="labelRotation">
       <property name="text">
        <string>Rotation</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="spinBoxRotX">
       <property name="minimum">
        <double>-360.0</double>
       </property>
       <property name="maximum">
        <double>360.0</double>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QDoubleSpinBox" name="spinBoxRotY">
       <property name="minimum">
        <double>-360.0</double>
       </property>
       <property name="maximum">
        <double>360.0</double>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QDoubleSpinBox" name="spinBoxRotZ">
       <property name="minimum">
        <double>-360.0</double>
       </property>
       <property name="maximum">
        <double>360.0</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
